### DISCUSSION TITLE
table使用异步请求数据，短期携带不同参数多次reloadData，最终响应慢的结果集渲染到页面

### DISCUSSION THREAD

**Discussion by youbins** [layui/layui#1708] [问答]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

疑是 BUG

### 使用版本

v2.8.7

### 问题描述

问题：
table使用异步请求数据，短期携带不同参数多次reloadData，最终响应慢的结果集渲染到页面。

期望：
不管多少次reloadData，table都应该显示最后一次请求的结果，而不是显示响应最慢的结果集。

PS：有什么办法中断上一次的请求，或者把异步请求方法开放出来，让用户可自定义请求方法？

### 业务代码

```auto
无
```

### 截图补充

无

### 浏览器

Microsoft Edge 版本 123.0.2420.41 (正式版本)

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by Sight-wcg** [comment]
这不是 BUG，你可以尝试用 jQuery ajax 的 beforeSend 回调，或者防抖解决此问题。

**Comment by youbins** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，您的邮件我已经收到，我会尽快给您回复。
